### ISSUE TITLE
relase link is still this github page

### ISSUE THREAD

**Issue by xarryon** [aakash4305/GBDF#1]
Thanks for this great work! Does this release link wrong ? Cuz I click the link but still stay in this github webpage.

**Comment by aakash4305** [comment]
Hi Yang, thanks for supporting our work. The dataset labels are available in the releases we couldn’t upload the original videos of the dataset due to copyright issue. So we provided labels of that videos from the original datasets which you can download easily with our labels file.

**Comment by xarryon** [comment]
I found it, thx !! Well, would you plan open source the test label file ?

**Comment by aakash4305** [comment]
Not sure, Because we are not working on this project anymore. Sorry for the inconvenience.

**Comment by aakash4305** [comment]
But creating test set would be very easy for you (provided the training set).

**Comment by zhugeshenren** [comment]
Thank you very much for your work, your dataset has helped us a lot.

**Comment by aakash4305** [comment]
You’re welcome.
